core/src/main/kotlin/com/tschuchort/compiletesting/CompilationResult.kt

```diff
@@ -0,0 +1,67 @@
+package com.facebook.buck.jvm.java.javax.com.tschuchort.compiletesting
+
+import com.tschuchort.compiletesting.KotlinCompilation
+import com.tschuchort.compiletesting.KotlinJsCompilation
+import com.tschuchort.compiletesting.listFilesRecursively
+import java.io.File
+import java.net.URLClassLoader
+import org.jetbrains.kotlin.compiler.plugin.ExperimentalCompilerApi
+
+/** Result of the compilation. */
+@ExperimentalCompilerApi
+sealed class CompilationResult(
```

Comment:
Could be `interface`. Don't think you gain anything by using a class here except maybe the future pain of the rigidity of inheritance.

Reply:
[54c624c](https://github.com/ZacSweers/kotlin-compile-testing/pull/136/commits/54c624c9503f431295730894e3f13a955e4398e2)